packages/riipen-ui/src/themes/default.js

```diff
@@ -112,6 +112,14 @@ export default {
   },
   spacing: factor => 5 * factor,
   transitions: {
+    create: (properties = [], options = {}) =>
+      properties
+        .map(
+          property =>
+            `${property} ${options.duraction || 300}ms ${options.easing ||
+              "cubic-bezier(0.4, 0, 0.2, 1)"}`
+        )
+        .join(", "),
```

Comment:
This is a pretty nice helper that Material-UI also provides so I thought I would "borrow" it.

Reply:
http://vignette4.wikia.nocookie.net/ronaldmcdonald/images/0/0f/Imgres.jpeg/revision/latest?cb=20150625050506